securemeet.py

```
@@ -0,0 +1,907 @@
+        self.transcript_text.config(state=tk.DISABLED)
+
+    # Audio processing loop -------------------------------------------------------
+    def process_audio_loop(self):
+        while not self.stop_processing.is_set():
+            data = self.audio_processor.get_audio_data()
+            if data is None:
+                time.sleep(0.05); continue
+            self.audio_buffer.append(data.copy())
+            diar = self.speaker_diarizer.process_audio(data, SAMPLE_RATE)
+            for seg in diar["segments"]:
+                start = seg["start"]; end = seg["end"]; spk = seg["speaker"]
```

Ruff (0.12.2)

Multiple statements on one line (semicolon)

(E702)